src/Swashbuckle.SwaggerGen/Annotations/SwaggerProducesResponseTypeAttribute.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Swashbuckle.SwaggerGen.Annotations
+{
```

Comment:
Purely a style preference, but I'm really not a big fan of XML comments, or comments in general for that matter. My philosophy is - strive for expressive code that reveals it's intention on it's own, reserving comments for hacks or non-intuitive workarounds that can't be expressed well with code alone. My bad for not having some contributor guidelines, but any chance you could remove the XML comments :)